release_notes/unreleased.md

```diff
@@ -1 +1,7 @@
 **Unreleased**
+**CSVIMPORT Release Notes - Published by Splunk December 25, 2021**
+
+
+**Version 2.0.0 - December 25, 2021**
+
+* Support for CSVIMPORT app [PAPP-7226]
```

Comment:
Use standard sentences that we are following in all the apps. App name is incorrect and doesn’t required here